src/corner_kick/src/index.html

```diff
@@ -0,0 +1,42 @@
+<!--
+    The HTML that loads when our application window opens.
+    Webpack will inject our application bundle in this file.
+-->
+
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Corner Kick</title>
+
+        <link rel="shortcut icon" type="image/png" href="../favicon.png" />
+
+        <meta charset="utf-8" />
+        <meta
+            name="viewport"
+            content="width=device-width, initial-scale=1, shrink-to-fit=no"
+        />
+
+        <!-- Icon library -->
+        <link
+            href="https://fonts.googleapis.com/icon?family=Material+Icons"
+            rel="stylesheet"
+        />
+    </head>
+
+    <body>
+        <!-- This is where our application is initially mounted -->
+        <div id="react"></div>
+
+        <!-- This where various UI elements are added using React portals-->
```

Comment:
Could we add a quick note here to the effect of `The order/arrangement of these elements should have no effect *whatsoever* on the layout of the actual applications, these are just "hangers" to allow us to get a reference on which to hang various components`

As another thought, why do we need multiple "hangers"? Could we not just have one hanger here to hang everything on? A quick comment elaborating on this would be good.

Reply:
Added comment. We need multiple hangers because we cannot define a complex layout using only one `div` element

Reply:
This makes a lot more sense with the layout being specified in `index.html`.